qldbdriver/statement_execution_test.go

```diff
@@ -448,76 +451,89 @@ func TestStatementExecution(t *testing.T) {
 		assert.True(t, ok)
 	})
 
-	t.Run("Execution metrics for stream result", func(t *testing.T) {
+	t.Run("Execution metrics", func(t *testing.T) {
 		driver, err := testBase.getDriver(ledger, 10, 4)
 		require.NoError(t, err)
 		defer driver.Shutdown(context.Background())
-		defer cleanup(driver, testTableName)
 
-		// Insert docs
-		_, err = driver.Execute(context.Background(), func(txn Transaction) (interface{}, error) {
-			return txn.Execute(fmt.Sprintf("INSERT INTO %s << {'col': 1}, {'col': 2}, {'col': 3} >>", testTableName))
-		})
-		require.NoError(t, err)
+		type TestTable struct {
+			Name string `ion:"Name"`
+		}
+		record := TestTable{singleDocumentValue}
 
 		selectQuery := fmt.Sprintf("SELECT * FROM %s as a, %s as b, %s as c, %s as d, %s as e, %s as f",
 			testTableName, testTableName, testTableName, testTableName, testTableName, testTableName)
 
-		_, err = driver.Execute(context.Background(), func(txn Transaction) (interface{}, error) {
-			result, err := txn.Execute(selectQuery)
+		insertDocuments := func(driver *QLDBDriver) {
+			_, err = driver.Execute(context.Background(), func(txn Transaction) (interface{}, error) {
+				return txn.Execute(fmt.Sprintf("INSERT INTO %s <<?, ?, ?>>", testTableName), record, record, record)
+
+			})
 			require.NoError(t, err)
+		}
 
-			for result.Next(txn) {
-				// IOUsage test
-				ioUsage := result.GetConsumedIOs()
-				require.NotNil(t, ioUsage)
-				assert.True(t, *ioUsage.GetReadIOs() > 0)
-
-				// TimingInformation test
-				timingInfo := result.GetTimingInformation()
-				require.NotNil(t, timingInfo)
-				assert.True(t, *timingInfo.GetProcessingTimeMilliseconds() > 0)
-			}
-			return nil, nil
-		})
-		assert.NoError(t, err)
-	})
+		t.Run("Execution metrics for stream result", func(t *testing.T) {
+			defer cleanup(driver, testTableName)
 
-	t.Run("Execution metrics for buffered result", func(t *testing.T) {
-		driver, err := testBase.getDriver(ledger, 10, 4)
```

Comment:
Indentation looks off? Is it just me?

Reply:
In Go, we use [tabs for indentations](https://golang.org/doc/effective_go#formatting). On Github,[ it displays tab characters as 8 spaces by default](https://github.com/tiimgreen/github-cheat-sheet#adjust-tab-space).

You can add `?ts=4` at the end of a URL to show the correct indentation. e.g. [communicator.go](https://github.com/awslabs/amazon-qldb-driver-go/blob/master/qldbdriver/communicator.go?ts=4)

On my Goland IDE, it shows the indentations correctly:
<img width="959" alt="Screen Shot 2021-02-23 at 1 37 19 PM" src="https://user-images.githubusercontent.com/57543366/108911565-41ac5500-75dc-11eb-872c-b5a7d39cccc5.png">